src/__tests__/index.test.ts

```diff
@@ -375,23 +375,14 @@ const vectors: Array<TestVector> = [
       { text: 'tz3NdTPb3Ax2rVW2Kq9QEdzfYFkRwhrQRPhX', hex: '0002193b2b3f6b8f8e1e6b39b4d442fc2b432f6427a8' },
     ],
   },
-  {
-    name: 'ADA',
```

Comment:
Why do you remove ADA?

Reply:
I must have confused "ADA" with "AVA" when transitioning the commit from Travala to Avalanche, my bad!